INSTANCES

```diff
@@ -0,0 +1,18 @@
+Template variables:
+
+IOCTOP   - The ioc/common directory for this IOC.
+TOP      - The ioc/xxx directory for this IOC, where xxx is the three-letter hutch.
+ENGINEER - The engineer responsible for this IOC.
+LOCATION - The location of this IOC.
```

Comment:
There seems to be a small desync between these and the variables in use in the `children/ioc-tdklambda.cfg` file. Which is correct?